pkg/quotes/enums.go

```diff
@@ -18,6 +18,7 @@ func (d DriverType) String() string {
 }
 
 var (
+	DriverIndex         = DriverType{"index"}
```

Comment:
Please, do not reintroduce `DriverIndex`. Index is not a driver, it's a runtime mode. If multiple drivers are configured, index will be started with all of them automatically.

Reply:
All in all, I begin to question if we need `Source` field in `TradeEvent` at all